games/f.yaml

```diff
@@ -618,6 +618,27 @@
   updated: 2022-09-01
   info: FreeCS is a free-software reimplementation of Counter-Strike 1.5 running on the FTEQW engine.
 
+- name: FreeHeroes
+  development: active
+  frameworks:
+  - Qt
+  images:
+  - https://github.com/mapron/FreeHeroes/raw/c84877ca37c10476182a755cb64f836725dd2f5c/docs/en/images/battle_emulator.png
+  - https://github.com/mapron/FreeHeroes/raw/c84877ca37c10476182a755cb64f836725dd2f5c/docs/en/images/battle_widget.png
+  - https://github.com/mapron/FreeHeroes/raw/c84877ca37c10476182a755cb64f836725dd2f5c/docs/en/images/hero_dialog.png
+  info: FreeHeroes is free engine for Heroes of Might and Magic 3.
+  langs:
+  - C++
+  licenses:
+  - MIT
+  content: commercial
+  originals:
+  - Heroes of Might and Magic III
+  repo: https://github.com/mapron/FreeHeroes
+  status: semi-playable
```

Comment:
semi-playable is a direct lie.
You can emulate battles, true, but that 1/1000 part of 'playing' the game.

Till I am not finished adventure part I strongly against it.

Reply:
Well, there are three categories (as far as I know):

- playable
- semi-playable
- unplayable

Parts of the game are limited playable. That's why I thought semi-playable was the most appropriate. But I can change it to "unplayable" if you like.